k8s/namespaces/ia/ia-task-configuration/ithc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-task-configuration
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.java: glob:pr-85-*
+    hmcts.github.com/prod-automated: disabled
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/ia/ia-task-configuration:prod-3bd0b248
```

Comment:
I can't remember but I guess Flux will replace this with the PR tag when the operator runs right?

Reply:
Correct. It just needs to exist. Otherwise, it will fail.

Reply:
nice one